pkg/backends/fake/backend_test.go

```diff
@@ -238,7 +260,28 @@ func TestCheckOrUpdateStatus(t *testing.T) {
 			return true, nil, &StatusError{}
 		})
 
-		backend := New(client, lt, zap.NewNop())
-		assert.Error(t, backend.CheckOrUpdateStatus(context.TODO()))
+		backend := &Backend{
+			kubeClient: client,
+			logger:     zap.NewNop(),
+		}
+		assert.Error(t, backend.SyncStatus(context.TODO(), lt, &lt.Status))
 	})
 }
+
+func TestTransformLoadTestSpec(t *testing.T) {
+	var distributedPods int32 = 1
+
+	spec := loadTestV1.LoadTestSpec{
+		Type:            "Fake",
+		Overwrite:       true,
+		DistributedPods: &distributedPods,
+		Tags:            loadTestV1.LoadTestTags{"team": "kangal"},
+	}
+
+	b := Backend{}
+	err := b.TransformLoadTestSpec(&spec)
+
+	assert.NoError(t, err)
```

Comment:
Does it make sense to run other asserts if this one failed? If not then
```suggestion
	require.NoError(t, err)
```

Reply:
Nope, I changed